README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+- While adaptive website targets devices with a fixed width, fully responsive website targets devices width a max-width or min-width.
```

Comment:
Please explain more

Reply:
Adaptive websites make use of fixed layout width with media queries, that target specific device widths and as such don't scale well on unaccounted device widths.

Responsive websites make use of min-width or max-width layout with media queries, that target ranges of device widths and thus, scale well across all device widths.